README.md

```diff
@@ -335,38 +419,104 @@ describe('my-elem', () => {
 });
 ```
 
-The filename and location of the resulting image will be based on the suite names and test name.
+### Configuring the Snapshot Area
 
-Sometimes you might be writing tests to verify a component's position within the whole viewport. For snapshots of the whole page, set up your components and pass `document` to the assertion:
+By default, the snapshot area will be a rectangle around the target element plus a `10px` buffer margin on each side. To use a different margin, pass it as an option:
 
 ```javascript
-import { fixture, html } from '@brightspace-ui/testing';
+await expect(elem).to.be.golden({ margin: 20 });
+```
 
-describe('my-elem', () => {
-  describe('situation1', () => {
-    it('state1', async() => {
-      await fixture(html`<my-dialog opened></my-dialog>`);
-      await expect(document).to.be.golden();
-    });
-  });
-});
+#### Capturing the Viewport
+
+To capture the entire viewport, pass `document` as the target element to the assertion:
+
+```javascript
+await expect(document).to.be.golden();
+```
+
+#### Changing the Vdiff Target
+
+If the target element doesn't accurately represent the area to be captured, the `vdiff-target` CSS class can be applied to the element that does.
+
+In the following example, the initial target element's area will not contain the nested element. By adding the `vdiff-target` CSS class to the nested element, it becomes the new vdiff target.
+
+```javascript
+const elem = await fixture(html`
+  <span>
+    <div class="vdiff-target" style="height: 50px; width: 30px;">hello</div>
+  </span>
+`);
+await expect(elem).to.be.golden();
```

Comment:
`vdiff-target` only works inside components, I'd expect this to just use the `span`'s size.  Maybe we switch this section with the other (since people outside `core` will probably need `vdiff-include` a lot more than `vdiff-target`), and then something like this?

```javascript
// example-component.js
render() {
  return html`
    <span>
      <div class="vdiff-target" style="height: 50px; width: 30px;">hello</div>
    </span>
  `;
}

// example-component.vdiff.js
const elem = await fixture(html`<example-component></example-component>`);
await expect(elem).to.be.golden();
```

Reply:
Makes sense, will adjust.